src/views.lisp

```diff
@@ -135,12 +142,12 @@
          (file-posts (select-posts (+ 1 *file-display-count*) (* page *file-display-count*) :type "file" :board board))
          (text-posts-next-page-p (> (length text-posts) *text-display-count*))
          (file-posts-next-page-p (> (length file-posts) *file-display-count*))
-         (stylized-text-posts (map 'list #'stylize-text-post (if text-posts-next-page-p
```

Comment:
If you know your input is a list, you can just use mapcar. Map is more for generic operations against things like arrays and strings.